src/instrumentsscene/view/instrumentspanelcontextmenumodel.cpp

```diff
@@ -83,11 +94,57 @@ void InstrumentsPanelContextMenuModel::loadItems()
         currentOrder.customized = false;
         m_orders.push_back(currentOrder);
     }
+}
```

Comment:
Now that the functionality of `loadItems()` has changed so much, I think it would be nice to rename it, for example to `loadInstrumentOrders()`.

Reply:
done :)